handler/core/src/test/java/io/knotx/fragments/handler/action/logging/InMemoryCacheActionFactoryLoggingTest.java

```diff
@@ -0,0 +1,208 @@
+package io.knotx.fragments.handler.action.logging;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.knotx.fragments.api.Fragment;
+import io.knotx.fragments.handler.action.InMemoryCacheActionFactory;
+import io.knotx.fragments.handler.api.Action;
+import io.knotx.fragments.handler.api.domain.FragmentContext;
+import io.knotx.fragments.handler.api.domain.FragmentResult;
+import io.knotx.junit5.KnotxExtension;
+import io.knotx.server.api.context.ClientRequest;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.VertxTestContext;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(KnotxExtension.class)
+class InMemoryCacheActionFactoryLoggingTest {
```

Comment:
All cases focus on the INFO log level. Can we check that there are no node log entries when the action (in-memory action) ends with _success transition and log level is ERROR.